vcbuild.bat

```diff
@@ -114,6 +114,7 @@ if "%i18n_arg%"=="full-icu" set configure_flags=%configure_flags% --with-intl=fu
 if "%i18n_arg%"=="small-icu" set configure_flags=%configure_flags% --with-intl=small-icu
 if "%i18n_arg%"=="intl-none" set configure_flags=%configure_flags% --with-intl=none
 if "%i18n_arg%"=="without-intl" set configure_flags=%configure_flags% --without-intl
+if "%engine%"=="chakracore" set configure_flags=%configure_flags% --without-intl
```

Comment:
If `"%i18n_arg%"=="without-intl"`, will this cause `%configure_flags% --without-intl` be set twice, and will that cause a problem?